app/models/manageiq/providers/ibm_power_vc/inventory/parser/cloud_manager.rb

```diff
@@ -1,2 +1,210 @@
 class ManageIQ::Providers::IbmPowerVc::Inventory::Parser::CloudManager < ManageIQ::Providers::Openstack::Inventory::Parser::CloudManager
+  def parse_vm(vm, hosts)
+    super
+    server = persister.vms.find_or_build(vm.id.to_s)
+
+    # saving processor type and amount
+    flavor = get_flavor(vm)
```

Comment:
Overall we need to confirm the flavor is the right place to pull this information from.  IIRC, flavors contain the requested value and there is other metadata on the VM object that provides the actual value for these type of settings.  In other words, we're not really interested in showing the flavor values - but rather the VM's actual values - so we need to get the data from the right spot.

Reply:
This is now changed as we are now picking the VM attributes.